### PR TITLE
Unpack frozensets as frozensets

### PR DESCRIPTION
**Pull request by klaussfreire**
Instead of unpacking them as subclasses of frozensets.
This matches other primitive types, is more interoperable with
code that expects frozensets, and prevents breaking singleton
properties of the frozensets (that empty frozensets unpack
as a single instance no matter where)

### CODE REVIEW

**Review by tzulberti-jampp — APPROVED** [review]

**Comment by klaussfreire** [comment]
Ping @GervasioCalderon @zeitos

File: sharedbuffers/_version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '0.4.2'
\ No newline at end of file
+__version__ = '0.4.3'
```
Comment (zeitos) [inline sharedbuffers/_version.py]: shouldn't this file have a new line?

Comment (klaussfreire) [reply sharedbuffers/_version.py]: It's autogenerated by setup.py, if it's missing we can make the change in setup.py to add the newline.

Comment (zeitos) [reply sharedbuffers/_version.py]: ok

**Review by zeitos — APPROVED** [review]

**Pull request merged**